extensions/interactions/LogicProof/static/js/completeSpec.js

```diff
@@ -16,7 +16,7 @@
  * @fileoverview Complete tests for LogicProof interaction JS.
  */
 
-var errorWrapper = function(
+errorWrapper = function(
```

Comment:
Should we move this into the describe block if it's not meant to be in global scope? Ditto for sharedErrorWrapper.

Reply:
Ah no, this needs to be in global scope. This was producing errors since the declaration for errorWrapper was different in different tests. I first moved it to globals but renaming it was a better option. This is fixed now.